packages/core/nuxt-theme-module/theme/components/CartSidebar.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div id="cart">
     <SfSidebar
-      :visible="isCartSidebarOpen"
+      :visible="true"
```

Comment:
I think this prop should default to `true`. WDYT @AdamPawlinski and @justyna-13 ?